sale_triple_discount/__manifest__.py

```diff
@@ -1,23 +1,22 @@
-# -*- coding: utf-8 -*-
 # Copyright 2015 ADHOC SA  (http://www.adhoc.com.ar)
-# Copyright 2017 Alex Comba - Agile Business Group
+# Copyright 2017 - 2019 Alex Comba - Agile Business Group
 # Copyright 2017 Tecnativa - David Vidal
 # Copyright 2018 Jacques-Etienne Baudoux (BCIM sprl) <je@bcim.be>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     'name': 'Sale Triple Discount',
-    'version': '10.0.1.0.1',
+    'version': '12.0.1.0.0',
     'category': 'Sales Management',
```

Comment:
Please change it to 'Sales' to appear correctly in OCA shop.

Reply:
Thanks @rousseldenis, changed in my last force pushed commit.